gems/aws-sdk-cloudfront/spec/cookie_signer_spec.rb

```diff
@@ -37,6 +37,21 @@ module CloudFront
           expect(cookie['CloudFront-Signature']).to eq('n4V7xum3wA-w1PaCMyEMpWVXdfw-Yt8I26RpZJKc~Nk8yQh8LYOxewItGJXFq0BxnKuSEKoEVYVTFEteFAGKXwhkbC7K~JfL83aroPbRagjyG-V9Y5wMLccBAzMj5nHXxjvjlOu541VUR-RlR0KK106HT4-Hp1c~nyOmXs4R5mU_')
           expect(cookie['CloudFront-Key-Pair-Id']).to eq('CF_KEYPAIR_ID')
         end
```

Comment:
nit-pick, one empty line between 2 suites?

Reply:
@cjyclaire  Thanks! I added it ➕